awx/settings/defaults.py

```diff
@@ -1046,4 +1047,12 @@
 # Host metrics cleanup - minimal interval between two cleanups in days
 CLEANUP_HOST_METRICS_INTERVAL = 30  # days
 # Host metrics cleanup - soft-delete HostMetric records with last_automation < [threshold] (in months)
-CLEANUP_HOST_METRICS_THRESHOLD = 12  # months
+CLEANUP_HOST_METRICS_SOFT_THRESHOLD = 12  # months
```

Comment:
nit: I don't know how this, `CLEANUP_HOST_METRICS_THRESHOLD`, is used, and I don't know the typical practice in AWX, but from general principles it does not seem like this variable name MUST change, right? You could add `HARD_THRESHOLD` and leave `CLEANUP_HOST_METRICS_THRESHOLD` for backwards compatibility, and you could still add `*_SOFT_THRESHOLD` if you make sure it's always the same as the old `CLEANUP_HOST_METRICS_THRESHOLD`. You could then drop the unadorned varname, `CLEANUP_HOST_METRICS_THRESHOLD`, later on. It is probably not that important in this case, but in general I would avoid changing the settings names when possible.

Reply:
I would really not recommend that. Keeping just "threshold" and "hard_threshold" looks like the hard one will subset of the current one. But it's not true and will be confusing. 
Also this PR is part of the same ADR where the constant was introduced and will be backported. 
So to your question, yes, this variable MUST change because it'll become confusing other way